cmd/mdatagen/templates/component_test.go.tmpl

```diff
@@ -53,6 +53,13 @@ func (aneh *assertNoErrorHost) ReportFatalError(err error) {
 	assert.NoError(aneh, err)
 }
 
+func TestREADMEmdPresent(t *testing.T) {
+	require.FileExists(t, "README.md")
+}
+
+func TestMetadataYamlPresent(t *testing.T) {
+	require.FileExists(t, "metadata.yaml")
+}
```

Comment:
Not sure I understand how this can happen otherwise since it is generated by that file. If anything we can have this as a requirement in our ci/cd.

Reply:
you're right. I'll drop it then.